### PR TITLE
[Preview] Added azureDefender addon to aks-preview

### PR DESCRIPTION
**Pull request by gal-ofir-ms**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [ ] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [ ] Have you run `python scripts/ci/test_index.py -q` locally?

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your PR is merged into master branch, a new PR will be created to update `src/index.json` automatically.  
The precondition is to put your code inside this repo and upgrade the version in the PR but do not modify `src/index.json`.

### CODE REVIEW

**Comment by gal-ofir-ms** [comment]
Anyone assist to why checks are failing? Doesnt seem to be related to my code (what is license check? and Its getting 403 trying to download blobs)

**Review by fengzhou-msft — APPROVED** [review]

File: src/aks-preview/HISTORY.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ Release History
 +++++
 * Add --enable-azure-rbac and --disable-azure-rbac in aks update
 * Support disabling local accounts
-
+* Add addon `azure-defender` to list of available addons under `az aks enable-addons` command
 0.5.11
```
Comment (fengzhou-msft) [inline src/aks-preview/HISTORY.md:10]:
```suggestion
* Add addon `azure-defender` to list of available addons under `az aks enable-addons` command

0.5.11
```

**Pull request merged**